modules/layers/src/solid-polygon-layer/solid-polygon-layer.js

```diff
@@ -139,21 +139,21 @@ export default class SolidPolygonLayer extends Layer {
       indices: {size: 1, isIndexed: true, update: this.calculateIndices, noAlloc},
       positions: {
         size: 3,
-        accessor: ['extruded', 'fp64'],
+        accessor: 'getPolygon',
         update: this.calculatePositions,
         noAlloc
       },
-      positions64xyLow: {size: 2, accessor: 'fp64', update: this.calculatePositionsLow},
+      positions64xyLow: {size: 2, update: this.calculatePositionsLow},
```

Comment:
For another PR: I think we should soon make it so that we always calculate 64 bit positions. They could be used to great effect in 32-bit shaders as well, and it could clean up this `positions*` mess in many layers.